src/index.js

```diff
@@ -2,38 +2,101 @@ const {
   BaseKonnector,
   requestFactory,
   log,
-  saveFiles,
-  addData
+  errors,
+  saveBills
 } = require('cozy-konnector-libs')
-const request = requestFactory({ cheerio: true })
-
-const baseUrl = 'http://books.toscrape.com'
+let request = requestFactory()
+const jar = request.jar()
+request = requestFactory({
+  //debug: true,
+  cheerio: true,
+  jar: jar
+})
+const moment = require('moment')
+moment.locale('fr')
 
+const baseUrl = 'https://www.boulanger.com/webapp/wcs/stores/servlet/'
+const loginUrl = baseUrl + 'BLAuthentication'
+const billsUrl =
+  baseUrl + 'BLAccountOrdersHistoryCmd?purchase=allYear&store=store-site'
 
 module.exports = new BaseKonnector(start)
 
 function start(fields) {
-  // The BaseKonnector instance expects a Promise as return of the function
-  return request(`${baseUrl}/index.html`).then($ => {
-    // cheerio (https://cheerio.js.org/) uses the same api as jQuery (http://jquery.com/)
-    // here I do an Array.from to convert the cheerio fake array to a real js array.
-    const entries = Array.from($('article')).map(article =>
-      parseArticle($, article)
-    )
-    return addData(entries, 'com.toscrape.books').then(() =>
-      saveFiles(entries, fields)
-    )
-  })
+  log('info', 'Authenticating ...')
+  return authenticate(fields.email, fields.password)
+    .then(getList)
+    .then(entries => {
+      return saveBills(entries, fields, {
+        timeout: Date.now() + 60 * 1000,
```

Comment:
We can allow more time for the connector to run. Maybe 4 minutes?

Reply:
Not need of a non default timeout here, i will remove it.

Reply:
Done here: https://github.com/konnectors/boulanger/pull/2/commits/28ac64910e3192ce78422b6f34eb483dfec1abfe